tensorflow/c/c_api.h

```diff
@@ -1662,6 +1662,29 @@ TF_CAPI_EXPORT extern TF_Buffer* TF_GetAllRegisteredKernels(TF_Status* status);
 TF_CAPI_EXPORT extern TF_Buffer* TF_GetRegisteredKernelsForOp(
     const char* name, TF_Status* status);
 
+// --------------------------------------------------------------------------
+// Server functionality.
+
+// Server.
+typedef struct TF_Server TF_Server;
+
+// Creates new server.
+TF_CAPI_EXPORT extern TF_Server* TF_NewServer(const void* proto,
+                                              size_t proto_len,
+                                              TF_Status* status);
+
+// Starts a server.
+TF_CAPI_EXPORT extern void TF_StartServer(TF_Server* server, TF_Status* status);
```

Comment:
Let's more closely follow the convention of most of the types in this file. So:

- `TF_NewServer`
- `TF_ServerStart`
- `TF_ServerStop`
- `TF_ServerJoin`
- `TF_DeleteServer`